setup.py

```diff
@@ -20,7 +20,7 @@
 
 
 install_requires = [
-    'marshmallow>=2.13.5,<3.0.0',
+    'marshmallow==3.5.0',
```

Comment:
We might want to make this a `>=` vs. `==` as this is used in client packages.

Reply:
good call. done